src/alfasim_sdk/_internal/constants.py

```diff
@@ -418,6 +418,17 @@ class PumpViscosityModel(Enum):
     AnsiHi2010 = "ansihi_2010"
 
 
+class PumpThermalEfficiencyModel(Enum):
+    """
+    Defines the ESP thermal efficiency model
+        - ``Constant``: An user-defined constant value is used to define to transform the ESP mechanical energy to heat.
+        - ``PumpEfficiencyCurve``: The ESP efficiency curve is used to transform the ESP mechanical energy to heat.
+    """
+
+    Constant = "constant"
+    PumpEfficiencyCurve = "pump_eff_curve"
```

Comment:
Is `pump_eff_curve` the best term? 
Do you think `pump_efficiency_curve` is to large?
What about `adjusted_efficiency_curve`, or `efficiency_curve_based`, or `efficiency_based`? (something in those lines)

Remember that this string will be present in alfacase files.

Reply:
Following your suggestion, I renamed it to `efficiency_curve_based`.